identity/data_user.go

```diff
@@ -0,0 +1,56 @@
+package identity
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/databrickslabs/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+
+// DataSourceUser returns information about user specified by user name
+func DataSourceUser() *schema.Resource {
+	type entity struct {
+		UserName             	string   `json:"user_name"`
+		Home             	string   `json:"home" tf:"computed"`
+		Alphanumeric             	string   `json:"alphanumeric" tf:"computed"`
+	}
+
+	s := common.StructToSchema(entity{}, func(
```

Comment:
By the way, if you feel that schema map is shorter - you can do it as well :) just a side thought

Reply:
Done.